implementations/rust/c/bindings/src/helpers.rs

```diff
@@ -7,3 +8,34 @@ pub fn ockam_error_is_none(error: &ockam_error_t) -> bool {
 pub fn ockam_error_has_error(error: &ockam_error_t) -> bool {
     error.code != 0
 }
+
+impl Default for ockam_vault_secret_t {
+    fn default() -> Self {
+        ockam_vault_secret_t {
+            attributes: ockam_vault_secret_attributes_t {
+                length: 0,
+                type_: ockam_vault_secret_type_t::OCKAM_VAULT_SECRET_TYPE_BUFFER,
```

Comment:
Here it's using type_ but elsewhere in the code, it's using xtype. Need to be consistent.

Reply:
The reason to that is that type cannot be used as a property name in rust, right? But this code is bindings from c, and c doesn't have that restriction. So why would we change property name in c code?